### PR TITLE
Fixies and translation.

### PR DESCRIPTION
**Pull request by lpj145**
More fixes and translation

### CODE REVIEW

File: pt/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Retornando dados e conjuntos de resultados
+Requisitando Dados e Conjuntos de Resultados
 ##########################################
```
Comment (markstory) [inline pt/orm/retrieving-data-and-resultsets.rst]: Headlines needs to be as long as the words, or our tooling won't build the docs.

**Comment by lpj145** [comment]
Dont merge this, i'm translating more, and after merge.

**Comment by markstory** [comment]
There is a merge conflict that you'll need to resolve. Generally the easiest way to do this is to merge `origin/3.0` into your changes and resolve the conflicts.

**Comment by andtxr** [comment]
This file needs a deep review. It is recommended to just submit a pull request when your translation is complete and mergeable.

@lpj145 would you like to join http://github.com/cakephp-br to join efforts?

**Comment by markstory** [comment]
Closing as this cannot be merged. If you can get it into a merge-able state, please re-open it.

**Pull request closed without merging**